cmds/cat/cat.go

```diff
@@ -25,12 +25,18 @@ var (
 	_ = flag.Bool("u", false, "ignored")
 )
 
-func catFile(w io.Writer, file string) error {
-	f, err := os.Open(file)
-	if err != nil {
-		return err
+func catFile(w io.Writer, file string) (err error) {
+	var f *os.File
+
+	if file == "-" {
```

Comment:
no no no no no. There is a name for stdin, and it's /dev/fd/0 or /dev/tty or something else depending on your kernel. We're not doing this, sorry :-)

Reply:
hm, the command roadmap includes the comment "Still need - for stdin", i felt encouraged ;)